packages/whale-api-client/__tests__/api/masternodes.test.ts

```diff
@@ -28,34 +33,36 @@ afterAll(async () => {
 describe('list', () => {
   it('should list masternodes', async () => {
     const data = await client.masternodes.list()
-    expect(Object.keys(data[0]).length).toStrictEqual(7)
+    expect(Object.keys(data[0]).length).toStrictEqual(9)
     expect(data.hasNext).toStrictEqual(false)
     expect(data.nextToken).toStrictEqual(undefined)
 
     expect(data[0]).toStrictEqual({
-      id: '03280abd3d3ae8dc294c1a572cd7912c3c3e53044943eac62c2f6c4687c87f10',
+      id: 'e86c027861cc0af423313f4152a44a83296a388eb51bf1a6dde9bd75bed55fb4',
+      sort: '00000000e86c027861cc0af423313f4152a44a83296a388eb51bf1a6dde9bd75bed55fb4',
       state: 'ENABLED',
-      mintedBlocks: 0,
-      owner: { address: 'bcrt1qyeuu9rvq8a67j86pzvh5897afdmdjpyankp4mu' },
-      operator: { address: 'bcrt1qurwyhta75n2g75u2u5nds9p6w9v62y8wr40d2r' },
+      mintedBlocks: expect.any(Number),
+      owner: { address: 'mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU' },
+      operator: { address: 'mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy' },
       creation: { height: 0 },
       resign: {
-        tx: '0000000000000000000000000000000000000000000000000000000000000000',
+        tx: '',
```

Comment:
```suggestion
        tx: undefined,
```

How about we make it optional, make it less misleading